src/main/resources/org/jenkinsci/plugins/workflow/job/Messages_fr.properties

```diff
@@ -0,0 +1,26 @@
+# The MIT License
+#
+# Copyright (c) 2021, Jenkins project contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+WorkflowJob.Description=Organise des activit&eacute;s de longue dur&eacute;e qui peuvent s''&eacute;tendre sur plusieurs \
```

Comment:
Please use Unicode characters (in ISO-8859-1 encoding, or `\uXXXX` escapes) rather than HTML entities.

Reply:
I've changed it, let me know if it's all right now.

Reply:
There should be no HTML (SGML, XML) entities anywhere outside of a `*.html` (`*.xml`) file. Best to look up documentation on how Java `*.properties` files work.